### ISSUE TITLE
[Bug] New content doesn't show posters

### ISSUE THREAD

**Issue by SonOfDiablo** [jurialmunkey/skin.arctic.zephyr.2#251]
Recently new content (TV Shows and Movies) doest show posters.

I have tried clearing textures db and thumbnails, as well as switch to another skin (Estuary, in which everything shows correctly) and back.

It was working fine, then all of a sudden stopped showing posters for newly added shows and movies.

Is this a known issue with a known fix?

I would be happy to provide more data if needed.

**Comment by drinfernoo** [comment]
If they're not showing, then they don't have art attached. That's not a skin issue, but an issue with your library.

**Comment by SonOfDiablo** [comment]
@drinfernoo but it works in other skins.. I would think that would point to an issue with the skin would it not?

**Comment by drinfernoo** [comment]
Have you added artwork locally, or scraped it using a library scraper? This skin shows artwork, as long as there is artwork attached to the items.

**Comment by SonOfDiablo** [comment]
I use Artwork Beef.
It older content (scraped the same way) shows correctly, only newly added has issues.

Could there be some type of limit on number of shows/movies the skin can handle?

**Comment by SonOfDiablo** [comment]
I thought a visual aid would help clear things up. 
As you can see the poster is not showed in the Shows View, but it shows in the Episodes View

![Shows](https://i.imgur.com/hnMC8j1.jpg)
![Episodes](https://i.imgur.com/6W0OEKr.jpg)

**Comment by drinfernoo** [comment]
Have you ensured that you *are* getting that art from Artwork Beef?

**Comment by SonOfDiablo** [comment]
Yes, we'll I mean, as I mentioned in the original post the poster actually shows when I switch to Estuary.

**Comment by mechevarria** [comment]
I noticed this issue too once I upgraded to Kodi 19. Random movies don't show posters and widgets don't pull posters at all for movies or TV Shows. All of this stuff is getting populated by PlexKodiConnect for me. And just like what @SonOfDiablo said, if I switch to Estuary everything shows up.

**Comment by mechevarria** [comment]
This bug has something to do with the [Artwork Beef](https://forum.kodi.tv/showthread.php?tid=258886) addon no longer working in Kodi 19. Apparently this plugin was a stopgap and it is no longer needed in Kodi 19 so it doesn't look like it will be ported.
